pkg/store/snapshot/device/store.go

```diff
@@ -16,7 +16,7 @@ package device
 
 import (
 	"context"
-	"errors"
+	"github.com/atomix/go-client/pkg/client/errors"
```

Comment:
should this be using `onos-lib-go/pkg/errors` instead? It think it probably amounts to the same thing

Reply:
Good catch. Fixed.

Reply:
Actually, we need both. The errors being returned by the store should be using `onos-lib-go/pkg/errors`, but the errors being received from the Atomix primitive are of the `atomix/go-client/pkg/errors` type. So, a check like `if errors.IsNotFound(err)` on an error returned from the Atomix primitive _has_ to use the Atomix `errors` package, and Atomix errors should be converted to ONOS errors using the ONOS `errors.FromAtomix` function. We certainly should be doing that everywhere there's an error so stores are not returning Atomix errors.

What you can

Reply:
Check out the way it's implemented in onos-e2sub:
https://github.com/onosproject/onos-e2sub/blob/master/pkg/store/subscription/store.go
Every error returned by Atomix is converted to an ONOS error with `errors.FromAtomix`.